bitmap-mosnter/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "bitmap-mosnter",
+  "version": "0.1.0",
+  "description": "bit-map project",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/andrewpeterwatson/lab-04-bitmap.git"
+  },
+  "author": "bitmap-monster",
```

Comment:
Changing this to the actual people involved in the project is good to track who's helping contribute